cirq-core/cirq/contrib/svg/svg_test.py

```diff
@@ -36,6 +36,8 @@ def test_validation():
     with pytest.raises(ValueError):
         circuit_to_svg(cirq.Circuit())
 
-    q0 = cirq.LineQubit(0)
-    with pytest.raises(ValueError):
-        circuit_to_svg(cirq.Circuit([cirq.Moment([cirq.X(q0)]), cirq.Moment([])]))
+
+def test_empty_moments():
```

Comment:
what about a test for a circuit that has non-empty moments and one empty moment? this is the common case after e.g. you run an optimizer to combine gates and it leaves behind empty moments

Reply:
Oh, that sounds like a good plan. Added.